sphinx_rtd_theme/versions.html

```diff
@@ -1,34 +1,50 @@
-{% if READTHEDOCS %}
+{%- block versions_menu %}
+{%- if theme_versions_menu|tobool %}
 {# Add rst-badge after rst-versions for small badge style. #}
   <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="{{ _('Versions') }}">
     <span class="rst-current-version" data-toggle="rst-current-version">
-      <span class="fa fa-book"> Read the Docs</span>
```

Comment:
@agjohnson @Blendify 

Trying to understand this PR - won't this effectively remove Read the Docs branding from the theme even when published on Read the Docs?

* if this is the case, then I think it's hard to merge the PR for 1.1 without an alternative.
* if this is NOT the case, then I don't see any issues here.

Reply:
It will be reinjected via our API, but then there will be side effects like something similar to a FOUC, where the Read the Docs branding reappears after the API call finishes.

I still feel like this whole issue requires a bit more thought. Our primary concern is users hosting docs on RTD, so this change is mostly outside our immediate concerns. If any user needs to remove the RTD branding, they can override `versions.html` now.

I'd say this is a good PR to come back to, after more discussion about goals here and goals for the flyout.